File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem23.py
# Electricity Bill Calculator
# Write a program that calculates electricity bill based on units consumed and rate per unit, then displays the total bill amount.


units_consumed = float(input('Enter the units consumed: '))
rate_per_unit = float(input('Enter rate per unit: '))

total_bill_amount = units_consumed*rate_per_unit
print(f'The total bill amount is: {total_bill_amount:.2f}')

File: Basic Programs/Problem_23.js
/*--------------------------------
Problem 23: Electricity Bill Calculator

Write a program that calculates electricity bill based on units consumed and rate per unit, then displays
the total bill amount.
--------------------------------*/

const unitsConsumed = 200;
const unitRate = 6;
const fixedCharge = 100;

const eleCharged = unitsConsumed * unitRate;

const totalBillAmt = eleCharged + fixedCharge;

console.log(
  `Unit Consumed: ${unitsConsumed}, Rate per unit: ${unitRate}, Fixed charged: ${fixedCharge}`,
);
console.log(
  `Electricity charged: ${eleCharged}, Total bill amount: ${totalBillAmt}`,
);

/* OUTPUT:
Unit Consumed: 200, Rate per unit: 6, Fixed charged: 100
Electricity charged: 1200, Total bill amount: 1300
*/
